src/lib.rs

```diff
@@ -63,20 +63,71 @@ use serde_json::Value;
 use std::{env, io, io::Read, iter, path::Path, time::Duration};
 use url::form_urlencoded;
 
+macro_rules! API_REFERENCE_URL {
+    () => {
+        "https://docs.docker.com/engine/api"
+    };
+}
+
+macro_rules! API_VERSION {
+    () => {
+        "v1.41"
+    };
+}
```

Comment:
These should be consts.
```suggestion
const API_REFERENCE_URL: &'static str = "https://docs.docker.com/engine/api";
const API_VERSION: &'static str = "v1.41";
```

Reply:
I couldn't make them consts because concat! macro that works on compile time only works with string literals. I've tried using consts but no luck.